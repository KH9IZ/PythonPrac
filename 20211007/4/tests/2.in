"exp(x)", "tan(x)", "x/(y**2 + 1)"
321
