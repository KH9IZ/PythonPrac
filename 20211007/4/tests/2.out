1.8445396458116389e+139
